src/site/content/en/learn/html/overview/index.md

```diff
@@ -46,7 +46,7 @@ HTML is very, very forgiving. For example, if we omit the closing `</li>` tags,
 </ul>  
 ```
 
-Although it is valid to not close an `<li>`, it isn't good practice.  The closing `</ul>` is mandatory. If the unordered list's closing tag is omitted, the browser will try to determine where your list and list items end, but you might not agree with the decision.
```

Comment:
The main point of this patch is to avoid saying "it isn't good practice" about something that in fact _is_ a good practice per Google's HTML style guide.